src/packages/eslint-plugin-eigenspace-react/README.md

```diff
@@ -0,0 +1,49 @@
+# CodeStyle [![Build Status](https://travis-ci.com/eigen-space/codestyle.svg?branch=master)](https://travis-ci.com/eigen-space/codestyle)
+
+Main language: `TypeScript`
```

Comment:
It seems that the description is copied. Need to fix it. The description should only apply to this project.